src/js/profile/application_settings.js

```diff
@@ -302,11 +302,14 @@ export class ApplicationSettings extends ReadWriteProxy {
         return 1;
     }
 
-    getMovementSpeed() {
+    /**
+     * @param {boolean} moveFaster
+     */
+    getMovementSpeed(moveFaster = false) {
         const id = this.getAllSettings().movementSpeed;
         for (let i = 0; i < movementSpeeds.length; ++i) {
             if (movementSpeeds[i].id === id) {
-                return movementSpeeds[i].multiplier;
+                return movementSpeeds[moveFaster && i + 1 < movementSpeeds.length ? i + 1 : i].multiplier;
```

Comment:
Not too sure about this, i think moveFaster should just be * 2 or so

Reply:
I was thinking that as a simpler way to handle it. The speed steps are actually doubled anyway. I'll just do 2 * in the next commit.